src/org.ts

```diff
@@ -73,3 +82,79 @@ export async function createOrg(
     result: { name: org.name },
   };
 }
+
+export async function updateOrg(
+  name: string,
+  orgData: OrgSchemaPatch,
+  options: ServiceOptions,
+): Promise<Result<undefined, OrgProblemType>> {
+  if (orgData.name !== undefined && name !== orgData.name) {
+    return {
+      didSucceed: false,
+      reason: OrgProblemType.INVALID_ORG_NAME,
+    };
+  }
+
+  const validationFailure = validateOrgData({ ...orgData }, options);
+
+  if (validationFailure !== undefined) {
+    return { didSucceed: false, reason: validationFailure };
+  }
+
+  const memberAccessType =
+    orgData.memberAccessType && MEMBER_ACCESS_TYPE_MAPPING[orgData.memberAccessType];
+
+  const orgModel = getModelForClass(OrgModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+  try {
+    await orgModel.updateOne(
+      {
+        name,
+      },
+      { ...orgData, memberAccessType },
+    );
+  } catch (err) {
+    throw err as Error;
+  }
+
+  options.logger.info({ name: orgData.name }, 'Org updated');
+  return {
+    didSucceed: true,
+  };
+}
+
+export async function getOrg(
+  name: string,
+  options: ServiceOptions,
+): Promise<Result<OrgSchema, OrgProblemType>> {
+  const orgModel = getModelForClass(OrgModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+  let org;
+
+  try {
+    org = await orgModel.findOne({
+      name,
+    });
+  } catch (err) {
+    throw err as Error;
+  }
+  if (org === null) {
+    return {
+      didSucceed: false,
+      reason: OrgProblemType.ORG_NOT_FOUND,
+    };
+  }
+
+  return {
+    didSucceed: true,
+
+    // TODO : implement a mapper that transforms an OrgSchema to the OrgSchemaType object
```

Comment:
mapper functions might be useful tools. example:
transform(OrgSchema) -> OrgSchemaType
transform2(OrgSchemaType) -> OrgSchema

Reply:
Agreed

Reply:
If we're not implementing that in this PR, which is totally fine, then let's please create an issue and delete this TODO. Otherwise, we'll just forget about it -- also, TODOs introduce a ESLint warnings.